prow/cmd/hook/main.go

```diff
@@ -75,6 +76,7 @@ func main() {
 		}
 		githubClient = github.NewFakeClient(*githubBotName)
 		githubClient.Logger = logrus.StandardLogger()
+		githubClient.SetTrustedOrgs(*trustedOrgs)
```

Comment:
That particular line is called when local is set and a fake github client is used.  The other one is for a standard run.  I'll consolidate both calls after the conditional block to simplify the code.